test/integ/pandas/test_pandas_tools.py

```diff
@@ -300,6 +300,21 @@ def test_invalid_table_type_write_pandas(
             )
 
 
+def test_empty_dataframe_auto_create_table_write_pandas(
```

Comment:
The test name seems redundant, the issue we fix is irrelevant to `auto_create_table`.